pkg/util/validation/validation_test.go

```diff
@@ -146,6 +146,39 @@ func TestValidateOverrideSpec(t *testing.T) {
 			},
 			expectError: true,
 		},
+		{
+			name: "overrideSpec.targetCluster.fieldSelector has unsupported key",
+			overrideSpec: policyv1alpha1.OverrideSpec{
+				TargetCluster: &policyv1alpha1.ClusterAffinity{
+					FieldSelector: &policyv1alpha1.FieldSelector{
+						MatchExpressions: []corev1.NodeSelectorRequirement{
+							{
+								Key:      "foo",
+								Operator: corev1.NodeSelectorOpIn,
+								Values:   []string{"fooCloud"},
+							}}}},
+			},
+			expectError: true,
+		},
+		{
+			name: "overrideRules.[index].targetCluster.fieldSelector has unsupported operator",
+			overrideSpec: policyv1alpha1.OverrideSpec{
+				OverrideRules: []policyv1alpha1.RuleWithCluster{
+					{
+						TargetCluster: &policyv1alpha1.ClusterAffinity{
+							FieldSelector: &policyv1alpha1.FieldSelector{
+								MatchExpressions: []corev1.NodeSelectorRequirement{
+									{
+										Key:      "foo",
+										Operator: corev1.NodeSelectorOpIn,
```

Comment:
Since this case is testing `unsupported operator`, we should give it a valid `key` and a invalid `operator`.